### PR TITLE
Fix IPv6 iptables.

### PR DESCRIPTION
**Pull request by myidpt**
Resolves step 1 in https://github.com/istio/istio/issues/5273

### CODE REVIEW

File: tools/deb/istio-iptables.sh [hunk 1]
```diff
@@ -210,6 +212,7 @@ if [ -n "${INBOUND_PORTS_INCLUDE}" ]; then
     table=nat
   fi
   iptables -t ${table} -N ISTIO_INBOUND
+  ip6tables -t ${table} -N ISTIO_INBOUND
   iptables -t ${table} -A PREROUTING -p tcp -j ISTIO_INBOUND
```
Comment (costinm) [inline tools/deb/istio-iptables.sh]: Pretty sure you need to do this for ip6 as well

**Comment by myidpt** [comment]
@costinm `ip6tables` is installed with `iptables`. But the command `ip6tables -t nat -N ISTIO_INBOUND` yields:
```
ip6tables v1.6.0: can't initialize ip6tables table `nat': Table does not exist (do you need to insmod?)
Perhaps ip6tables or your kernel needs to be upgraded.
```
This indicates the kernel module `ip6table_nat` is not enabled. I'm trying to enable it using `modprobe` to see if it works. But if the kernel doesn't have the module at all, things will get messy. Maybe the only way is to install the kernel image or change the container image. I haven't verified either works yet.

Do you have any suggestions?

**Comment by myidpt** [comment]
@costinm PTAL

**Comment by myidpt** [comment]
@costinm Unfortunately the following command doesn't work on the kernel I tested. Did some research and found it might be because the IPv6 `state` module is not available on the kernel.
```
+ ip6tables -A INPUT -m state --state ESTABLISHED -j ACCEPT
ip6tables: Protocol wrong type for socket.
```
I'll move forward to have `|| true` following the command so that when the `state` module is not available, reject all INPUT traffic including ESTABLISHED.

**Comment by myidpt** [comment]
@costinm Costin, do you want to take a final look before I check this in?

**Pull request merged**